modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/HeadersSideBar.scala

```diff
@@ -27,26 +44,38 @@ import scala.concurrent.duration._
 object HeadersSideBar {
   final case class Props(model: ModelProxy[HeaderSideBarFocus]) {
     def canOperate: Boolean = model().status.canOperate
-    def sequenceObserver: Either[Observer, SequenceObserverFocus] = model().observer
+    def selectedObserver
+      : Either[Observer, Either[DayCalObserverFocus, SequenceObserverFocus]] =
```

Comment:
Ugly type to have the options:
* Default observer
* Daycal observer
* instrument observer